src/amp-story-player/amp-story-player-impl.js

```diff
@@ -177,6 +178,54 @@ export class AmpStoryPlayer {
     this.layoutCallback();
   }
 
+  /**
+   * Adds stories to the player, both adding new anchor tags to the player,
+   * and creating or assigning iframes to those that are close to the
+   * current playing story.
+   *
+   * @param {!Array<!{href: string, title: ?string}>} stories
+   */
+  addStories(stories) {
+    for (let i = 0; i < stories.length; i++) {
+      const story = stories[i];
+      const anchor = this.createStoryAnchor_(story);
```

Comment:
Please add some code to validate what you're getting in this `stories` parameter, they might send wrong data

Reply:
Would checking its an Array work? i.e. `Array.isArray(stories)` or would it be better to check the structure of the objects too? i.e. `Array.isArray(stories) && stories.every((e) => typeof e === 'object' && e !== null && e.href)`

Also, what should happen if the data is wrong? throw? return silently?